src/components/side_panel/chart/gauge_chart_panel/gauge_chart_design_panel.ts

```diff
@@ -139,6 +145,12 @@ export class GaugeChartDesignPanel extends Component<Props, SpreadsheetChildEnv>
   }
 
   updateSectionRule(sectionRule: SectionRule) {
+    const invalidValueReasons = this.checkSectionRuleFormulasAreValid(this.state.sectionRule);
+    if (invalidValueReasons.length > 0) {
+      this.state.sectionRuleDispatchResult = new DispatchResult(invalidValueReasons);
```

Comment:
Would it be possible to consider changing `sectionRuleDispatchResult` to something else like 
`sectionRuleErrors`? Creating a fake dispatch result just to be iable to use the ChartTerms looks so wrong :'(